types/committee_id.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/binary"
+	"sort"
+)
+
+type CommitteeID [32]byte
+
+func (cid CommitteeID) MessageIDBelongs(msgID MessageID) bool {
+	id := msgID.GetDutyExecutorID()[16:]
+	return bytes.Equal(id, cid[:])
+}
+
+// Return a 32 bytes ID for the cluster of operators
+func GetCommitteeID(committee []OperatorID) CommitteeID {
```

Comment:
This should be used when we create the MessageID. 
Currently in spec we just take the `Identifier` from `Controller` for CommitteeRunner

However, I do believe it will be better to follow the pattern that we use in other runners where we create the `MessageID` in `ProcessConsensus`
It wasn't done this way because it was unclear how to get to the DomainType in `Share`.

The following solutions were discussed:
1. Using the first share (similar to leading validator). This was in code but looked hacky.
2. Moving DomainType to `CommitteeMember` (what about PreconMessages though)
3. Having it both in `Share` and `CommitteeMember` (bad duplicate)
4. `DomainType` should be in some global config since a node should be in one network. We can have it derived like in the Eth Spec (my favorite...)

Reply:
@GalRogozinski which solution we choose, please also open an issue for point 4.

Reply:
Going with option 3

Reply:
option 4 already exists in implementation on `NetworkConfig`. IMO 3 doesn't make sense because what you said about a domain for whole network.

Reply:
@y0sher 
maybe speak with @lior-blox?

Reply:
#439